driver.sh

```diff
@@ -22,6 +22,13 @@ setup_variables() {
         shift
     done
 
+    set +u
+    if [[ -z "$ARCH" ]]; then
```

Comment:
Could do `if [[ -z "${ARCH:-}" ]]; then` instead of the `set +u` dance.

Reply:
`./driver.sh: line 25: ARCH: unbound variable`
am I holding it wrong? 🤡

Reply:
maybe default to zero and compare against a value of zero, rather than use `-z`?

Reply:
Yes, that is probably a better idea.

Reply:
done